.gitignore

```diff
@@ -74,7 +74,7 @@ web_modules/
 .yarn-integrity
 
 # dotenv environment variable files
-.env
+.env*
```

Comment:
This would ignore all examples.

Reply:
example is named `example.env`

Reply:
> I like it. However, I would prefer the JS version for better usability and cross compatibility.

This is not feasible, people have trouble installing and configuring docker, `nvm` / `npm` is an unreasonable requirement for stack